lib/openhab/dsl/items/item_command.rb

```diff
@@ -54,7 +55,7 @@ def item_state(command_enum, methods = {})
           command_enum.values.each do |command|
             status_method = "#{command.to_s.downcase}?"
             status_method = methods.transform_keys(&:to_sym).fetch(status_method.to_sym, status_method)
-            logger.trace("Creating status method (#{status_method}) for #{self.class}")
+            logger.trace("Creating status method (#{status_method}) for #{self}")
```

Comment:
Just curious (no comment on the change), was the .class just not needed?

Reply:
yeah. `self` is already `DimmerItem`, etc. here. so with `self.class`, the logging was coming out `Creating status method (on?) for Class`, which seemed not useful.